### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: android/assets/ui/languages.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
     <Language id="en" name="English" fontSet="xolonium-kwajong"/>
     <Language id="fr" name="Français" fontSet="xolonium-kwajong"/>
     <Language id="de" name="Deutsch" fontSet="xolonium-kwajong"/>
+    <Language id="nl" name="Dutch" fontSet="xolonium-kwajong"/>
```
Comment (agateau) [inline android/assets/ui/languages.xml:5]: I added this entry for Dutch to appear in the UI, but the value of the `name` attribute must be the *translated name* of the language. Can you edit this file to replace "Dutch" here, or tell me how to say Dutch in Dutch :)

**Review by agateau — CHANGES_REQUESTED** [review]

File: android/assets/ui/languages.xml (hunk 1 above)
Comment (Vistaus) [reply android/assets/ui/languages.xml:5]: Dutch in Dutch = **_Nederlands_** 🙂

**Review by agateau — APPROVED** [review]

File: android/assets/ui/languages.xml (hunk 1 above)
Comment (agateau) [reply android/assets/ui/languages.xml]: Thanks! Merging this.

Comment (Vistaus) [reply android/assets/ui/languages.xml]: Thank you for merging! 🙂

**Pull request merged**